staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -197,7 +197,7 @@ func (s *EtcdOptions) addEtcdHealthEndpoint(c *server.Config) {
 	c.HealthzChecks = append(c.HealthzChecks, healthz.NamedCheck("etcd", func(r *http.Request) error {
```

Comment:
we're switching the etcd health check to make an actual etcd request with a real client in https://github.com/kubernetes/kubernetes/pull/65027 to address #64909